packages/cinterop/build.gradle.kts

```diff
@@ -378,30 +379,31 @@ fun Task.buildSharedLibrariesForJVM() {
         if (copyJvmABIs) {
             // copy files (Linux)
             project.file("src/jvmMain/linux-build-dir/librealmc.so")
-                .copyTo(project.file("src/jvmMain/resources/jni/linux/librealmc.so"), overwrite = true)
+                .copyTo(project.file("$jvmJniPath/linux/librealmc.so"), overwrite = true)
             genHashFile(platform = "linux", prefix = "lib", suffix = ".so")
 
             // copy files (Windows)
             project.file("src/jvmMain/windows-build-dir/Release/realmc.dll")
-                .copyTo(project.file("src/jvmMain/resources/jni/windows/realmc.dll"), overwrite = true)
+                .copyTo(project.file("$jvmJniPath/windows/realmc.dll"), overwrite = true)
             genHashFile(platform = "windows", prefix = "", suffix = ".dll")
         }
     }
 
-    outputs.file(project.file("src/jvmMain/resources/jni/macos/librealmc.dylib"))
-    outputs.file(project.file("src/jvmMain/resources/jni/macos/dynamic_libraries.properties"))
+    inputs.dir(project.file("$absoluteCorePath/src"))
```

Comment:
Do we need to use the entire folder as input or would just using `realm.h` be enough? 🤔

Reply:
If we don't set the whole tree then it won't trigger cmake's incremental reduild when the capi is stable but other sources have changed